MonoHbb/MonoHbbQuantities.py

```diff
@@ -132,8 +132,8 @@ def getBins(quant):
                 low='70.'
                 high='110.'
             elif 'Wmass' in quant:
-                bins='110'
-                low='50.'
+                bins='32'
```

Comment:
why this?

Reply:
Just to see how the distribution looks in the entire range of (0,160).